tests/runner.py

```diff
@@ -1294,47 +1294,72 @@ class BrowserCore(RunnerCore):
   MAX_UNRESPONSIVE_TESTS = 10
 
   unresponsive_tests = 0
+  responsive_tests = 0
 
   def __init__(self, *args, **kwargs):
     super(BrowserCore, self).__init__(*args, **kwargs)
 
   @classmethod
   def setUpClass(cls):
     super(BrowserCore, cls).setUpClass()
-    cls.also_asmjs = int(os.getenv('EMTEST_BROWSER_ALSO_ASMJS', '0')) == 1
-    cls.port = int(os.getenv('EMTEST_BROWSER_PORT', '8888'))
+    BrowserCore.also_asmjs = int(os.getenv('EMTEST_BROWSER_ALSO_ASMJS', '0')) == 1
```

Comment:
Is there any particular reason why we are using `BrowserCore` explicitly everywhere instead of `cls`?

Reply:
We use `BrowserCore.*` elsewhere to refer to these same properties (where `cls` does not arrive as a parameter, as they aren't class methods). I feel that having it consistent in usage everywhere is simpler and less bug prone?

Reply:
IIUC its better to use `cls` when you have it as an argument.   Works like `self`.